games-engines/scummvm/scummvm-1.9.0.recipe

```diff
@@ -56,41 +58,56 @@ BUILD_REQUIRES="
 	devel:libogg$secondaryArchSuffix
 	devel:libpng$secondaryArchSuffix
 	devel:libSDL$secondaryArchSuffix
+	#devel:libsdl_net$secondaryArchSuffix #not needed yet
 	devel:libtheora$secondaryArchSuffix
 	devel:libvorbis$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
-"
+	"
 
 BUILD_PREREQUIRES="
 	cmd:gcc$secondaryArchSuffix
-	cmd:git
 	cmd:ld$secondaryArchSuffix
-	cmd:libtoolize
+	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
 	cmd:nasm
 	cmd:pkg_config$secondaryArchSuffix
-"
+	"
 
 BUILD()
 {
 	libtoolize --force --copy --install
 	CPPFLAGS=`freetype-config --cflags` ./configure --prefix=$prefix \
-	--bindir=$appsDir --libdir=$libDir --datarootdir=$dataDir \
-	--mandir=$manDir --docdir=$docDir \
-	--disable-debug --enable-release #--disable-all-engines
-	#--disable-all-engines can be used to only compile scummvm without
-	#engines, which makes it build faster, those that are still under
-	#construction can be built with --enable-all-engines for testing purposes
-
+		--bindir=$appsDir --libdir=$libDir --datarootdir=$dataDir \
+		--mandir=$manDir --docdir=$docDir \
+		--disable-debug --enable-release --disable-eventrecorder \
+		#--disable-all-engines #can be used to compile scummvm without \
+		#engines, which makes it build faster (to test), those that \
+		#are still under construction can be built with --enable-all-engines \
+		#for testing purposes
 	make $jobArgs
 }
 
 INSTALL()
 {
 	make install
 	mv $appsDir/scummvm $appsDir/ScummVM
-	addResourcesToBinaries $portDir/additional-files/scummvm.rdef \
+
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		$portDir/additional-files/scummvm.rdef > scummvm.rdef
```

Comment:
@fbrosson I got the emails from you over github in regards to my master branch (but I can't see them here in the list) :)
To start with, I have one checkout "haikuports" where I do my work in, then I have a checkout for "haikuports_games" where copy the files that have been changed into before pushing them to git.
My master on "haikuports_games" isn't uptodate I know, and as you mentioned  that was probably what went wrong here.
I'll try to update with the guideliness you mentioned in the email, I just checked  your first line:
both give me the same output:

> ~/haikuports> git remote -v | grep haikuports/haikuports
> origin  https://github.com/haikuports/haikuports.git (fetch)
> origin  https://github.com/haikuports/haikuports.git (push)

> ~/haikuport_games> git remote -v | grep haikuports/haikuports
> origin  https://github.com/haikuports/haikuports.git (fetch)
> origin  https://github.com/haikuports/haikuports.git (push)